libs/blocks/locui-create/locui-create.js

```diff
@@ -0,0 +1,15 @@
+import { createTag } from '../../utils/utils.js';
+
+export default function init(el) {
+  el.classList.add('missing-details');
+  const heading = createTag('h2', null, 'Missing project details');
+  const paragraph = createTag('p', null, 'The project details were removed after you logged in. To resolve this:');
+  const steps = createTag('ol', null);
+  const goTo = createTag('p', { class: 'goto-step' }, 'Go to ');
+  const goToAnchor = createTag('a', { href: 'https://milostudio.adobe.com', target: '_blank', rel: 'noopener noreferrer' }, 'MiloStudio');
```

Comment:
Should we consider lower environments as well?

Reply:
Hi @narcis-radu ,
It will be to consider lower environments as there is no way to identify from which env the workflow is triggered as after successful msal login the query params get removed. So for placeholder we have kept for milostudio.adobe.com. If required we can remove the link and keep the text only.

Reply:
If I start a workflow on milostudio.dev.adob.com and click on the anchor, I might end up on milostudio.adobe.com where the workflow is not visible. Is my understanding correct?

Reply:
Yes, your understanding is correct.

Reply:
Hi @narcis-radu , 

We have added lower environments urls also in the instructions so that the user can click on the respective url to navigate to milostudio. Please have a look and let us know if it is fine by you.

Thanks